### PR TITLE
plugins/neocord: init

### PR DESCRIPTION
**Pull request by redyf**
Add support for https://github.com/IogaMaster/neocord

### CODE REVIEW

**Comment by redyf** [comment]
@traxys Hi traxys, I made PR for this plugin on nixpkgs and it was merged already. It shows up in nixpkgs search but only for the unstable branch. Is that why the checks failed?
```nix
 error: attribute 'neocord' missing
at /nix/store/sv7l2h04qc6k0yg3mikx58id5wql195y-source/plugins/utils/neocord.nix:14:51:
13| enable = mkEnableOption "neocord";
14| package = helpers.mkPackageOption "neocord" pkgs.vimPlugins.neocord;
```

**Comment by traxys** [comment]
If the PR recently tricked to nixos-unstable we first need to update the flake.lock

**Comment by traxys** [comment]
You should rebase onto main, we recently updated the flake.lock

File: tests/test-sources/plugins/utils/neocord.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  };
+
+  defaults = {
+    plugins.neocord = {
+      enable = true;
+
+      # General options.
+      autoUpdate = true;
+      logo = "auto";
+      logo_tooltip = null;
```
Comment (traxys) [inline tests/test-sources/plugins/utils/neocord.nix:13]: That would be logoTooltip

Comment (redyf) [reply tests/test-sources/plugins/utils/neocord.nix:13]: Thanks! I was looking for why this error was being caused for a while now hahaha

**Pull request closed without merging**